apps/root/src/hooks/earn/useFilteredStrategies.ts

```diff
@@ -5,23 +5,27 @@ import { searchByStrategyData } from '@common/utils/earn/search';
 import { StrategiesTableVariants } from '@state/strategies-filters/reducer';
 import { getComparator } from '@common/utils/earn/parsing';
 import { EarnPosition, Strategy } from 'common-types';
+import { StrategyColumnConfig } from '@pages/earn/components/strategies-table/components/columns';
 
-const isEarnPosition = (obj: Strategy | EarnPosition): obj is EarnPosition => 'strategy' in obj;
+const isEarnPosition = (obj: Strategy | EarnPosition[], variant: StrategiesTableVariants): obj is EarnPosition[] =>
```

Comment:
this doesn't actually use the obj right? just the variant

Reply:
It's used for the type assertion, so I can get rid of the union type and differenciate one case from the other